tests/classifiers/test_admin.py

```diff
@@ -30,3 +34,23 @@ def test_change_default_frequency(app: DjangoTestApp):
     form['is_default'] = True
     form.submit()
     assert list(Frequency.objects.filter(is_default=True)) == [another_frequency]
+
+
+@pytest.mark.django_db
+def test_change_default_status(app: DjangoTestApp):
+    Status.objects.all().delete()
+
+    admin = User.objects.create_superuser(email="admin@gmail.com", password="test123")
+    default_status = StatusFactory(is_default=True)
+    another_status = StatusFactory(is_default=False)
+    app.set_user(admin)
+    form = app.get(
+        reverse("admin:vitrina_classifiers_status_change", args=[another_status.pk])
+    ).forms["status_form"]
+    form["is_default"] = True
+    form["name"] = "Test"
+
+    form.submit()
+
+    assert list(Status.objects.filter(is_default=True)) == [another_status]
+    assert list(Status.objects.filter(is_default=False)) == [default_status]
```

Comment:
Let's add a docstring to explain how this work exactly, since reading this test caused some confusion.

If I understand correctly, we have this:

default_status which is default (`is_default=True`) and another_status, which is not the default (`is_default=False`) at the beginning. Then, we call an endpoint, to change the status of `another_status`, which makes it change from `False` to `True`. And because we need to have at least one status that is `is_default=True`, the change to `default_status` changes automatically?

Since now it is not clear, why `default_status` has changed automatically.

Additionally, I suggest this:

```suggestion
    assert Status.objects.filter(is_default=True).first() == another_status
    assert Status.objects.filter(is_default=False).first() == default_status
```

And asserting the `count` if you need it separately.